scripts/mkcloud

```diff
@@ -1338,6 +1339,9 @@ Optional
         If set, does not use the --insecure flag in openstack CLI commands.
     want_monasca_tsdb (Cloud9+ only)
         Allows setting time-series DB used for Monasca [influxdb|cassandra].
+    want_ipv6 (Currently in development)
+        Prepare crowbar to use a full IPv6 single stack control plane. Enabling will also set
+        the firmware_type to UEFI as it's required to netboot nodes in IPv6.
```

Comment:
Replace 'in' with 'over' -- I think that reads better.

Reply:
done